Gulpfile.js

```diff
@@ -417,7 +417,15 @@ task("generate-types-map", generateTypesMap);
 const cleanTypesMap = () => del("built/local/typesMap.json");
 cleanTasks.push(cleanTypesMap);
 
-const buildOtherOutputs = parallel(buildCancellationToken, buildTypingsInstaller, buildWatchGuard, generateTypesMap);
+const builtLocalDiagnosticMessagesGeneratedJson = "built/local/diagnosticMessages.generated.json";
+const copyBuiltLocalDiagnosticMessages = () => src(diagnosticMessagesGeneratedJson)

```

Comment:
Why this action exists shouldn't be tribal knowledge - could you include a comment on the task as to why we do the copy (so it's not removed by some future engineer looking to cull defunct build tasks, as it _looks_ defunct, since it's not copied into the LKG or otherwise used by anything in this repo).

Reply:
Fair - that's probably why it got removed in the first place.